src/cascade/dismod/serialize.py

```diff
@@ -442,15 +423,13 @@ def covariate_multiplier_iter(context):
         for rate_mul in rate.covariate_multipliers:
             yield rate_mul, "rate_value", rate
 
-    # β
-    for val_integrand in context.outputs.integrands:
-        for val_mul in val_integrand.value_covariate_multipliers:
-            yield val_mul, "meas_value", val_integrand
-
-    # γ
-    for std_integrand in context.outputs.integrands:
-        for std_mul in std_integrand.std_covariate_multipliers:
-            yield std_mul, "meas_std", std_integrand
+    for integrand in context.integrand_covariate_multipliers.values():
+        # β
```

Comment:
I put this unicode character in here. Think we're safe with those? Recommendation is not to use them b/c they still sometimes get misinterpreted.

Reply:
As comments or as variable names? I did once write a chunk of code that used fancy unicode variable names. It was kind of neat because I could almost copy and paste from math articles and expect it to run but they're too hard to type. No idea about misinterpretation.